From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Oleg <balbes-150@yandex.ru>
Date: Tue, 26 Oct 2021 08:52:40 +0200
Subject: [ARCHEOLOGY] fix firefly-rk3399 (#3227)

> X-Git-Archeology: - Revision 103ddb1adb9d3aba8ebb420762ac91d137c143c5: https://github.com/armbian/build/commit/103ddb1adb9d3aba8ebb420762ac91d137c143c5
> X-Git-Archeology:   Date: Tue, 26 Oct 2021 08:52:40 +0200
> X-Git-Archeology:   From: Oleg <balbes-150@yandex.ru>
> X-Git-Archeology:   Subject: fix firefly-rk3399 (#3227)
> X-Git-Archeology:
> X-Git-Archeology: - Revision 5fc2d639889e35319d9f51b15129cf96b2d80ab8: https://github.com/armbian/build/commit/5fc2d639889e35319d9f51b15129cf96b2d80ab8
> X-Git-Archeology:   Date: Tue, 26 Oct 2021 22:19:01 +0200
> X-Git-Archeology:   From: Piotr Szczepanik <piter75@gmail.com>
> X-Git-Archeology:   Subject: Moved missing firefly-rk3399 u-boot patch to a new location
> X-Git-Archeology:
> X-Git-Archeology: - Revision 320bf63430b442700cf1cea7e290b9ef7bdde0e9: https://github.com/armbian/build/commit/320bf63430b442700cf1cea7e290b9ef7bdde0e9
> X-Git-Archeology:   Date: Mon, 15 Nov 2021 12:46:38 +0100
> X-Git-Archeology:   From: Oleg <balbes-150@yandex.ru>
> X-Git-Archeology:   Subject: Added usb boot support for NanoPC T4 (#3256)
> X-Git-Archeology:
> X-Git-Archeology: - Revision 2813365dd25e3ad110936cbf014b95b38d7090ec: https://github.com/armbian/build/commit/2813365dd25e3ad110936cbf014b95b38d7090ec
> X-Git-Archeology:   Date: Mon, 07 Nov 2022 21:29:00 +0100
> X-Git-Archeology:   From: Igor Pecovnik <igorpecovnik@users.noreply.github.com>
> X-Git-Archeology:   Subject: Move known non working rockhip64 boards to previous boot loader (#4392)
> X-Git-Archeology:
---
 configs/firefly-rk3399_defconfig | 9 +++++++++
 1 file changed, 9 insertions(+)

diff --git a/configs/firefly-rk3399_defconfig b/configs/firefly-rk3399_defconfig
index fe1c019f1d..a123e817d8 100644
--- a/configs/firefly-rk3399_defconfig
+++ b/configs/firefly-rk3399_defconfig
@@ -43,6 +43,8 @@ CONFIG_ETH_DESIGNWARE=y
 CONFIG_GMAC_ROCKCHIP=y
 CONFIG_NVME_PCI=y
 CONFIG_PCI=y
+CONFIG_PHY_ROCKCHIP_INNO_USB2=y
+CONFIG_PHY_ROCKCHIP_TYPEC=y
 CONFIG_PMIC_RK8XX=y
 CONFIG_REGULATOR_PWM=y
 CONFIG_REGULATOR_RK8XX=y
@@ -60,11 +62,18 @@ CONFIG_USB_EHCI_HCD=y
 CONFIG_USB_EHCI_GENERIC=y
 CONFIG_USB_DWC3=y
 CONFIG_USB_DWC3_GENERIC=y
+CONFIG_ROCKCHIP_USB2_PHY=y
+CONFIG_USB_KEYBOARD=y
+CONFIG_USB_GADGET=y
 CONFIG_USB_HOST_ETHER=y
 CONFIG_USB_ETHER_ASIX=y
 CONFIG_USB_ETHER_ASIX88179=y
 CONFIG_USB_ETHER_MCS7830=y
 CONFIG_USB_ETHER_RTL8152=y
 CONFIG_USB_ETHER_SMSC95XX=y
+CONFIG_DM_VIDEO=y
+CONFIG_DISPLAY=y
+CONFIG_VIDEO_ROCKCHIP=y
+CONFIG_DISPLAY_ROCKCHIP_HDMI=y
 CONFIG_SPL_TINY_MEMSET=y
 CONFIG_ERRNO_STR=y
-- 
Armbian

